eCourse_backend/users/views.py

```diff
@@ -32,6 +32,7 @@ def user_administration(request):
 @login_required
 @permission_required('users.manage_users', raise_exception=True)
 def create_lecturer_iframe(request):
```

Comment:
I don't like how this is done at all from a security standpoint, but we're running out of time, so it's fine

Reply:
There probably are also functions for this that are provided by the backend, but I don't think it could get much more secure from the Frontend

Reply:
Of course it's a catastrophe regarding security but at least it works